WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ItemDetails/ShippingLabelCustomsFormItemDetailsViewModel.swift

```diff
@@ -14,6 +14,14 @@ final class ShippingLabelCustomsFormItemDetailsViewModel: ObservableObject {
     ///
     let weightUnit: String
 
+    /// Flags if the Customs are under the EU description rules.
+    ///
+    private let shouldEnforceEUCustomsDescription: Bool
+
+    /// Flags if the Customs info tooltip should be presented.
+    ///
+    private let isEUTooltipAvailable: Bool
```

Comment:
I was a little confused by this property until I looked at `canDisplayTooltipInfoIcon`, because I thought it was meant to control whether the tooltip was displayed. Maybe it's worth updating the documentation to something other than "should be presented"? Maybe something like:

> Flags if the Customs info tooltip can be presented for this item.

Reply:
True! It looks confusing indeed 😅 I updated the documentation here: https://github.com/woocommerce/woocommerce-ios/pull/9740/commits/27aa1f4dcc6fe5200bf7f5fdcce16ce58a004bb5. Let me know what you think, thanks!

Reply:
That's much clearer, thanks!